src/closer-core.coffee

```diff
@@ -700,6 +700,26 @@ core =
         core.group_$_by(core.identity, coll))
      )
 
+  'not_$_every_$QMARK_' : (pred,coll) ->
+    assertions.arity 2, arguments.length
+    assertions.function pred
+    assertions.collection coll
+    !m.every? pred,coll
```

Comment:
Yes thought to do that first, but for assertions wrote it separately. Will change it now